eng/common/tools.ps1

```diff
@@ -764,6 +757,8 @@ function MSBuild-Core() {
     }
   }
 
+  Enable-Nuget-EnhancedRetry
```

Comment:
Is `MSBuild-Core` called without `MSBuild` L701 being called? It *feels* like this code path will call `Enable-Nuget-EnhancedRetry` twice?

Reply:
No idea, this came as part of whatever arcade bump I hit.